src/index.ts

```diff
@@ -449,17 +467,25 @@ export const plugin: Plugin<[FlexibleContainerOptions?], Root> = (options) => {
    * control weather has closing wither ":::" or not (check completeness)
    *
    */
-  function analyzeClosingNode(node: Paragraph): AnalyzeResult["flag"] {
+  function analyzeClosingNode(node: Paragraph, fenceLen: number): AnalyzeResult["flag"] {
```

Comment:
will receive fence instead of fenceLen, according to proposal of getOpeningFence